storage/ndb/rest-server/data-access-rondb/src/db-operations/pk/common.cpp

```diff
@@ -296,46 +281,41 @@ RS_Status SetOperationPKCol(const NdbDictionary::Column *col, NdbOperation *oper
     }
 
     // operation->equal expects a zero-padded char string
-    char pk[CHAR_MAX_SIZE_IN_BYTES];
+    primary_key      = new char[CHAR_MAX_SIZE_IN_BYTES];
```

Comment:
Could be a good idea to avoid large objects on stack as I found today. But there are still better alternatives to allocating an object for temporary storage. E.g. a pool of them or something like that.